src/governance-vote.h

```diff
@@ -68,6 +68,8 @@ enum vote_signal_enum_t  {
     VOTE_SIGNAL_CUSTOM20   = 35
 };
 
+static const int MAX_SUPPORTED_VOTE_SIGNAL = 4;
```

Comment:
Just because something is handy doesn't mean it's a good idea to use it.  I don't think aliasing enum symbols is something that is commonly done so I think most C++ programmers would expect different enum symbols to have distinct values.  Breaking that assumption can probably lead to bugs in certain cases (though I can't think of an example at the moment).

What I can do is change the line to:

`static const int MAX_SUPPORTED_VOTE_SIGNAL = VOTE_SIGNAL_ENDORSED`

though really I think this is a non-issue.  I can't think of any reason we would want to reassign the first 4 values.